5_evaluate.R

```diff
@@ -0,0 +1,143 @@
+source('5_evaluate/src/eval_utility_fxns.R')
+source('4_visualize/src/plot_data_utility_fxns.R')
+
+p5 <- list(
+  ##### Evaluate GCM model output #####
+  
+  ##### Evaluate NLDAS model output #####
+  
+  ### Prep predictions and observations
```

Comment:
I'd suggest using R's nesting headers capability by writing this as
```suggestion
  ###### Prep predictions and observations ######
```
so that this is considered a subheader when someone looks at the list of sections in RStudio.